geo_features.js

```diff
@@ -34,6 +35,9 @@ function featureStyle(feature) {
     strokeWeight = 1.5;
     strokeDash   = '1';
     fillColor    = '#a06060';
+  } else if(entity1name == 'Native American' && entity2type == 'people') {
+    strokeOn     = false;  // set to 'false' to hide territory 'boundary' rectangles; set to 'true' to visualize territory 'boundaries'
+    fillOn       = true;
```

Comment:
this is still causing the rectangles to fill in my view. I think what you want here is `fillOpacity = 0.0;` then it renders blankly. I tried `fillOn = false;` but then you lose the highlighting all together (most notably you lose the ability to pop-up feature information). But with fillOpacity = 0, and fillOn = true (the default anyway), it still fills with a gray box when moused over. This is due to line 142 in `ohmec.js` below, which sets the opacity to 0.7 when moused over. Normally that distinction is useful, but here highlights the gray rectangle.

I'm not sure the best way to convey this information to ohmec.js, but the following works:

line 23 of this file:

```javascript
  // declare a new feature 'borderless' that is false for most
  feature.borderless = false;
```

then overridden for Native (or whatever name we settle on)

```javascript
  } else if(entity1name == 'Native American' && entity2type == 'people') {
    strokeOn     = false;  // set to 'false' to hide territory 'boundary' rectangles
    fillOpacity  = 0.0;
    feature.borderless = true;
```

then in ohmec.js line 142:

```javascript
  if (layer.feature.geometry.type !== "Point") {
    let opacity = (layer.feature.borderless) ? 0.0 : 0.7;
    layer.setStyle({
      weight: 5,
      color: '#666',
      dashArray: '',
      fillOpacity: opacity
    });
```

Reply:
Great suggestions. I got this working as you described, except that I don't get a gray box when mousing over the tribe names. Candidly, I think this is just fine since the territories are ambiguous and certainly not rectangles anyway. So, mousing over the territories brings up the desired pop-up feature without revealing the underlying rectangle. I'm not sure why this is different for you, but let's see what happens when I push my code after I get through all of your other comments.

Reply:
Yeah, let's review that together offline. Things should be identical on both our sides; if not something is fishy.

Reply:
We're in synch